src/main/java/ooo/oshi/foreign/windows/Kernel32Library.java

```diff
@@ -60,5 +66,29 @@ public static int getCurrentProcessId() {
 
     private static final MethodHandle getCurrentProcessId = methodHandle("GetCurrentProcessId",
             FunctionDescriptor.of(JAVA_INT));
+    
+    /**
+     * Retrieves the NetBIOS name of the local computer. This name is established at system startup, when the system
+     * reads it from the registry.
+     *
+     * @return the NetBIOS name of the local computer.
+     */
+    public static String getComputerName() {
+    	try {
+        	SegmentAllocator allocator = SegmentAllocator.implicitAllocator();
+    		Addressable buffer = allocator.allocate(WinNT.MAX_COMPUTERNAME_LENGTH + 1);
+    		Addressable size = allocator.allocate(JAVA_LONG, WinNT.MAX_COMPUTERNAME_LENGTH + 1);
```

Comment:
Looking at [the method signature](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getcomputernamew) the size is a pointer to a 32-bit DWORD, so this should be JAVA_INT.